modules/platform/nuxeo-automation/nuxeo-automation-test/src/test/java/org/nuxeo/ecm/automation/server/test/AsyncOperationAdapterTest.java

```diff
@@ -222,6 +222,19 @@ public void testAsyncBulkAction() throws Exception {
         assertEquals("foo", getTitle(folder));
     }
 
+    @Test
+    public void testFailingBulkAction() throws Exception {
+        String result = async.newRequest(BulkRunAction.ID) //
+                             .set("action", AutomationBulkAction.ACTION_NAME)
+                             .set("query", "SELECT * FROM Folder")
+                             .set("bucketSize", "10")
+                             .set("batchSize", "5")
+                             .set("parameters", MAPPER.writeValueAsString("{}"))
```

Comment:
Can we just put `"{}"` without calling the MAPPER? `writeValueAsString` is used to serialize an object to JSON (or other), here you're giving to it a String. My first thought is: writing a string as string will return the input string. If it's not the case, having the final value in the test will help us to understand what makes it fail and what we want to control/catch.